examples/SampleUtils.cpp

```diff
@@ -81,6 +81,75 @@ namespace utils {
         return prod;
     }
 
+    const ml::Operator createActivationOperator(std::vector<SHARED_DATA_TYPE>& constants,
+                                                const ml::GraphBuilder& builder,
+                                                FusedActivation activation,
+                                                const std::vector<float>& options) {
+        ml::Operator activationOperator;
+        if (activation == FusedActivation::RELU) {
+            activationOperator = builder.ReluOperator();
+        } else if (activation == FusedActivation::RELU6) {
+            ml::ClampOptions clampOptions;
+            float min = 0;
+            auto minConstant = std::make_shared<std::vector<char>>(sizeof(float));
+            std::memcpy(minConstant->data(), &min, sizeof(float));
+            constants.push_back(minConstant);
+            float max = 6;
+            auto maxConstant = std::make_shared<std::vector<char>>(sizeof(float));
+            std::memcpy(maxConstant->data(), &max, sizeof(float));
+            constants.push_back(maxConstant);
+            clampOptions.minValue =
+                utils::BuildConstant(builder, {}, minConstant->data(), sizeof(float));
+            clampOptions.maxValue =
+                utils::BuildConstant(builder, {}, maxConstant->data(), sizeof(float));
+            activationOperator = builder.ClampOperator(&clampOptions);
+        } else if (activation == FusedActivation::SIGMOID) {
+            activationOperator = builder.SigmoidOperator();
+        } else if (activation == FusedActivation::LEAKYRELU) {
+            ml::LeakyReluOptions leakyReluOptions;
+            leakyReluOptions.alpha = options[0];
+            activationOperator = builder.LeakyReluOperator(&leakyReluOptions);
+        } else {
+            assert(0);
+        }
+        return activationOperator;
+    }
+
+    const ml::Operand createActivationOperand(std::vector<SHARED_DATA_TYPE>& constants,
+                                              const ml::GraphBuilder& builder,
+                                              const ml::Operand& input,
+                                              FusedActivation activation,
+                                              const std::vector<float>& options) {
```

Comment:
Could this method be merged into `createActivationOperator`? They share much code.

Reply:
Yes,  but I need to use these two functions to return different varibles: operator and operand, it's not easy to merge.

Reply:
How about use parameters?
```c++
template<class T>
T createActivation(std::vector<SHARED_DATA_TYPE>& constants,
                   const ml::GraphBuilder& builder,
                   const ml::Operand& input = {},
                   FusedActivation activation = FusedActivation::NONE,
                   const std::vector<float>& options = {});
```
Does it work?

Reply:
Thanks! I tried this way but faild...

Reply:
The failure might be caused by not all the paths can return the type T.

Then how about use pointers? like
```c++
void createActivation(std::vector<SHARED_DATA_TYPE>& constants,
                      const ml::GraphBuilder& builder,
                      FusedActivation activation,
                      void* activation,
                      const ml::Operand& input = {},
                      const std::vector<float>& options = {});
```

The usage would be
```c++
// create operator when there is no input
createActivation(constants, builder, FusedActivation::RELU, &(options.activation));

// create operand when there is input
createActivation(constants, builder, fusedActivation::RELU, &operand, input);
```

Reply:
Thanks, I will try.